### PR TITLE
Awesome Book using ES6 Feature 

### PR DESCRIPTION
**Pull request by melashu**
# In this project I have done the following tasks 

- Separate the class into a different module 
- I used the default export in each class 
- I used Luxon.js to get the current date and time
-  I change every method to arrow function

### CODE REVIEW

File: modules/BStorage.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return book;
+  };
+
+  static addBook = (book) => {
+    const bookList = BStorage.getBook();
+    bookList.push(book);
+    localStorage.setItem('ourbook', JSON.stringify(bookList));
+  };
+
+  static removeBook = (e) => {
```
Comment (muneebulrehman) [inline modules/BStorage.js:20]: If I add multiple books, go to the `List` page, remove all books, go to the `Add New` page, without adding any book and reloading, go back to the `List` page, I can still see some of the books even though I removed all the books. Consider fixing this issue.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @melashu , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️

- [ ] Kindly rename your default branch to main or master to be in accordance with the GitHub flow.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/BStorage.js (hunk 1 above)
Comment (melashu) [reply modules/BStorage.js:20]: @muneebulrehman  Thank you for your feedback.

Comment (melashu) [reply modules/BStorage.js:20]: I didn't understand why It worked fine sometimes

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @melashu ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job fixing the bug when removing books

 ## Required Changes ♻️

- [ ] You still have to rename your branch to `main` as requested by the previous reviewer

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by melashu** [comment]
> # Hi @melashu ✋,
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨
> 
> ## Highlights ⚡
> * [x]  Good job fixing the bug when removing books
> 
> ## Required Changes ♻️
> * [ ]  You still have to rename your branch to `main` as requested by the previous reviewer
> 
> _Check the comments under the review._
> 
> ## Optional suggestions 📂
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@vechicin  Thanks again. But he give me an optional comment which is why I didn't change it.

**Review by vechicin — APPROVED** [review]
# Hi @melashu ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job fixing the branch name

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**